x-pack/plugins/canvas/public/expression_types/arg_types/series_style/simple_template.js

```diff
@@ -44,7 +44,7 @@ export const SimpleTemplate = props => {
       ) : (
         <Fragment>
           <EuiFlexItem grow={false}>
-            <label>Color&nbsp;</label>
+            <label htmlFor={'series-style'}>Color&nbsp;</label>
```

Comment:
Strings don't need to be interpreted, this should be `htmlFor="series-style"`.

It would be nice if you also fixed the id on `ColorPickerMini` below too.